ssv/spectest/tests/runner/preconsensus/invalid_then_quorum.go

```diff
@@ -32,7 +32,6 @@ func InvalidThenQuorum() tests.SpecTest {
 				OutputMessages: []*types.SignedPartialSignatureMessage{
 					testingutils.PreConsensusContributionProofMsg(ks.Shares[1], ks.Shares[1], 1, 1), // broadcasts when starting a new duty
 				},
-				ExpectedError: "failed processing sync committee selection proof message: invalid pre-consensus message: could not verify Beacon partial Signature: wrong signature",
```

Comment:
shouldn't the error happen on the first quorum?

Reply:
The first invalid signature is overwritten by the correct one before a quorum is reached. Once the quorum is reached, all signatures stored are correct and no error is created